foreign_cc/private/framework/toolchains/windows_commands.bzl

```diff
@@ -41,6 +41,9 @@ def disable_tracing():
 def mkdirs(path):
     return "mkdir -p " + path
 
+def rm_rf(path):
+    return "rm -rf " + path
```

Comment:
Is this valid for windows? I see some places using this without going through this abstraction so maybe it is?

Reply:
Yes - its running in a msys2 environment so rm -rf is fine.